src/database/seeds/006_anotacao.js

```diff
@@ -0,0 +1,18 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('anotacao').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('anotacao').insert([
+        {
+          id_paciente: 2,
+          dt_hr_anotacao: "2021-09-07T21:56:30",
+          descricao: "bla bla bla",
+          anexo: "",
+          created_at: null, 
+          updated_at: null 
```

Comment:
Deletar estas linhas caso você concorde em retirar esses atributos da tabela Paciente.
```suggestion
          anexo: ""
```